client/lib/domains/test/assembler.js

```diff
@@ -41,15 +41,15 @@ describe( 'assembler', () => {
 			expired: undefined,
 			expirySoon: undefined,
 			googleAppsSubscription: undefined,
-			hasPrivacyProtection: undefined,
+			hasPrivacyProtectionProduct: undefined,
 			isAutoRenewing: undefined,
 			isPendingIcannVerification: undefined,
 			isPendingWhoisUpdate: undefined,
 			isPrimary: false,
 			name: DOMAIN_NAME,
```

Comment:
I think the unit tests are failing because the `redirectDomainObject` mock object is missing a prop:
 `privacyIncludedByDefault: undefined,`